build_scripts/es_setup/es_setup.sh

```diff
@@ -41,7 +41,25 @@ chmod 777 -R /data
 # 5. Start Elasticsearch
 sudo /etc/init.d/elasticsearch start
 
-# 6. Add the following lines to /etc/sudoers file
+# 6. Default setup for searchguard plugin
+sudo /etc/init.d/elasticsearch stop
+sudo cp /vagrant/build_scripts/es_setup/search-guard-5.zip /tmp/
```

Comment:
I think you can do these 4 lines in one line:
sudo unzip /vagrant/build_scripts/es_setup/search-guard-5.zip -z /usr/share/elasticsearch/plugins/

Reply:
Copying straight to the /usr/share/elasticsearch/plugins/ directory will not work when running build scripts. The command will say that it ran successfully but when you check after it did not actually unzip. I found that unzipping into /tmp/ first and then copying it over was the solution.